indico/modules/events/timetable/controllers/manage.py

```diff
@@ -224,14 +227,17 @@ def _process_PATCH(self, data):
             data['references'] = self._get_references(references)
 
         data['person_link_data'] = {v['person_link']: v['is_submitter'] for v in data.pop('person_links', [])}
-        
+
         with (track_time_changes(), track_location_changes()):
             update_contribution(self.contrib, data)
-        
+
         return ContributionSchema(context={'event': self.event}).jsonify(self.contrib)
 
     def _process_GET(self):
-        return ContributionSchema(context={'event': self.event}).jsonify(self.contrib)    
+        return ContributionSchema(context={'event': self.event}).jsonify(self.contrib)
+
+    def _process_DELETE(self):
+        delete_contribution(self.contrib)
```

Comment:
I recommend doing a `return '', 204` (that's "no content") at the end of any process functions that don't have a return value.